src/AddItems.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import { db } from './lib/firebase';
+import { capitalize } from './lib/utility';
+
+function AddItems(props) {
+  const [groceryItem, setGroceryItem] = useState('');
+  const [userToken, setUserToken] = useState('randomTokenOne');
+  const [itemFreq, setItemFreq] = useState(7);
```

Comment:
I don't think setting the state to `7` is working as expected on page load:

![image](https://user-images.githubusercontent.com/10378331/114867350-4f9b7c80-9dba-11eb-8582-d18ed50c660f.png)

Reply:
I see what you mean, the preview link is not checking the first radio button on page load. It's odd because it's working in the local environment. I think we can fix this using the useEffect hook but I'm curious to know why the difference in behavior between local and the preview link.

Reply:
It's probably because in order to set a radio item to be checked by default, you may need to add `checked` prop to the input, similar to:

```
<input type="radio" checked={itemFreq === 7} value={7} />
```

Reply:
Sounds good, I'll wait and see if it has resolved after preview link gets updated!

Reply:
@luisaugusto do you know how long it usually takes for the preview link to update? For some reason it isn't showing our changes from last night either, the 'Grocery List' view should show an unordered list but it's still showing a strigified JSON.

Reply:
Hey @kkatoch89 it might be because there is a merge conflict, would you be able to resolve that and then see if that triggers the deploy?